planner/core/physical_plans.go

```diff
@@ -917,8 +917,10 @@ type PhysicalUnionAll struct {
 type AggMppRunMode int
 
 const (
+	// NoMpp means the default value which runs not in MPP
+	NoMpp AggMppRunMode = iota
```

Comment:
What is it used for?

Reply:
if with this, `Mpp1Phase` is used as the default value, which is confused when it works as the default value.